diff --git a/node_modules/@internationalized/number/dist/types.d.ts b/node_modules/@internationalized/number/dist/types.d.ts
index 4fc8b86..80625ef 100644
--- a/node_modules/@internationalized/number/dist/types.d.ts
+++ b/node_modules/@internationalized/number/dist/types.d.ts
@@ -10,6 +10,8 @@ export class NumberFormatter implements Intl.NumberFormat {
     format(value: number): string;
     formatToParts(value: number): Intl.NumberFormatPart[];
     resolvedOptions(): Intl.ResolvedNumberFormatOptions;
+    formatRange(startDate: number | bigint, endDate: number | bigint): string;
+    formatRangeToParts(startDate: number | bigint, endDate: number | bigint): Intl.NumberFormatPart[];
 }
 /**
  * A NumberParser can be used perform locale aware parsing of numbers from Unicode strings,
